blocks/carousel/carousel.css

```diff
@@ -471,6 +527,11 @@
 }
 
 @media (width >= 64rem) {
+    .carousel-wrapper {
+        max-width: 75rem;
```

Comment:
Is there a reason for this? I think blocks should be automatically constrained by the parent section width.

Reply:
So in the existing website, IMO they were updating the same using javascript. Anyhow I have figure out a way to achieve the same experience without it. hence removed this